ocaml/testsuite/tests/typing-layouts/modules_alpha.ml

```diff
@@ -7,6 +7,7 @@ type t_any   : any
 type t_value : value
 type t_imm   : immediate
 type t_imm64 : immediate64
+type t_float64 : float64
```

Comment:
As with basics_alpha, I think there should basically be no changes in this file.